decidim-core/app/controllers/decidim/user_timeline_controller.rb

```diff
@@ -28,13 +29,18 @@ def activities
           organization: current_organization,
           resource_type: "all",
           scopes: current_user.interested_scopes,
-          follows: follows
+          follows: follows,
+          resource_name: filter.resource_type
```

Comment:
in the `default_filter_params` you're adding a `resource_type` param, but here you're passing `resource_name` to the search. Is this correct? why use two different names?

Reply:
Yes, that's correct. The reason is that `resource_type` was already in use on `ActivitySearch`; so the attribute that has the same name was causing a conflict in the code and the filter was having an incorrect behavior. That's why I wrote `resource_name:` instead of `resource_type:`